### PR TITLE
Related to Issue #2000. Updated Encoding parameter for Export-PSSession

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Related to Issue #2000 
Updated Encoding parameter for Export-PSSession
Style updates
Paragraph reflows
Fixed links

Acrolinx scores
Original version: 86
Updated version: 90

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[ab875cc](https://github.com/davidsmatlak/PowerShell-Docs/pull/3605/commits/ab875cc937a6fa9547833ad6fdabe5ae0ad2554b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-3.0&branch=pr-en-us-3605) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-4.0&branch=pr-en-us-3605) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-5.0&branch=pr-en-us-3605) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-5.1&branch=pr-en-us-3605) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-6&branch=pr-en-us-3605) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901242328590472-3605%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ltKF8ckCHp73Vcv65iuXgXmifw1Sg2YpouTi%2FrDaLKI%3D&st=2019-01-24T23%3A38%3A02Z&se=2019-02-24T23%3A43%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 1]
```diff
@@ -207,18 +223,19 @@ Accept wildcard characters: True
...
-- Script: Script blocks in the current session.
+- Alias. All PowerShell aliases in the current session.
+- All. All command types. It is the equivalent of `Get-Command -Name *`.
+- Application. All files other than PowerShell files in paths listed in the Path environment
+  variable (`$env:path`), including .txt, .exe, and .dll files.
+- Cmdlet. The cmdlets in the current session. Cmdlet is the default.
+- ExternalScript. All .ps1 files in the paths listed in the Path environment variable
+  (`$env:path`).
+- Filter and Function. All PowerShell functions.
+- Script. Script blocks in the current session.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: Accepted values: Alias, All, Function, Filter, Cmdlet, ExternalScript, Application, Script, **Workflow**

Add the 'Accepted values:` to the YAML block and add Workflow to the list above.

File: reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 2]
```diff
@@ -197,18 +222,20 @@ Accept wildcard characters: True
...
+
+- Alias. All PowerShell aliases in the current session.
+- All. All command types. It is the equivalent of `Get-Command -Name *`.
+- Application. All files other than PowerShell files in paths listed in the Path environment
+  variable (`$env:path`), including .txt, .exe, and .dll files.
+- Cmdlet. The cmdlets in the current session. Cmdlet is the default.
+- ExternalScript. All .ps1 files in the paths listed in the Path environment variable
+  (`$env:path`).
+- Filter and Function. All PowerShell functions.
+- Script. Script blocks in the current session.
```
Comment (sdwheeler) [inline reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: The accepted values list is correct but Configuration and Workflow are missing from the list above.

File: reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 3]
```diff
@@ -218,25 +244,20 @@ Accept wildcard characters: False
...
+
+- Alias. All PowerShell aliases in the current session.
+- All. All command types. It is the equivalent of `Get-Command -Name *`.
+- Application. All files other than PowerShell files in paths listed in the Path environment
+  variable (`$env:path`), including .txt, .exe, and .dll files.
+- Cmdlet. The cmdlets in the current session. Cmdlet is the default.
+- ExternalScript. All .ps1 files in the paths listed in the Path environment variable
+  (`$env:path`).
+- Filter and Function. All PowerShell functions.
+- Script. Script blocks in the current session.
```
Comment (sdwheeler) [inline reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: The accepted values list is correct but Configuration and Workflow are missing from the list above.

File: reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 4]
```diff
@@ -219,25 +245,20 @@ Accept wildcard characters: False
...
+
+- Alias. All PowerShell aliases in the current session.
+- All. All command types. It is the equivalent of `Get-Command -Name *`.
+- Application. All files other than PowerShell files in paths listed in the Path environment
+  variable (`$env:path`), including .txt, .exe, and .dll files.
+- Cmdlet. The cmdlets in the current session. Cmdlet is the default.
+- ExternalScript. All .ps1 files in the paths listed in the Path environment variable
+  (`$env:path`).
+- Filter and Function. All PowerShell functions.
+- Script. Script blocks in the current session.
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md]: The accepted values list is correct but Configuration and Workflow are missing from the list above.

File: reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 5]
```diff
@@ -219,25 +245,20 @@ Accept wildcard characters: False
...
+
+- Alias. All PowerShell aliases in the current session.
+- All. All command types. It is the equivalent of `Get-Command -Name *`.
+- Application. All files other than PowerShell files in paths listed in the Path environment
+  variable (`$env:path`), including .txt, .exe, and .dll files.
+- Cmdlet. The cmdlets in the current session. Cmdlet is the default.
+- ExternalScript. All .ps1 files in the paths listed in the Path environment variable
+  (`$env:path`).
+- Filter and Function. All PowerShell functions.
+- Script. Script blocks in the current session.
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md]: The accepted values list is correct but Configuration and Workflow are missing from the list above.

File: reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the Accepted values to YAML and added Workflow to the list with brief description based on about_Workflows.

File: reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the Accepted values to YAML and added Configuration and Workflow to the list with with links to about_Session_Configurations and about_Workflows.

File: reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md (hunk 3 above)
Comment (davidsmatlak) [reply reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the Accepted values to YAML and added Configuration and Workflow to the list with with links to about_Session_Configurations and about_Workflows.

File: reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md (hunk 4 above)
Comment (davidsmatlak) [reply reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the Accepted values to YAML and added Configuration and Workflow to the list with with links to about_Session_Configurations and about_Workflows.

File: reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md (hunk 5 above)
Comment (davidsmatlak) [reply reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md]: Updated the Accepted values YAML to alpha order and added Configuration and Workflow to the list with with links to about_Session_Configurations (v5.1) and about_Workflows.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[faca359](https://github.com/davidsmatlak/PowerShell-Docs/pull/3605/commits/faca359aa06c0549cd8fad9c89c067022f5b050a)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[faca359](https://github.com/davidsmatlak/PowerShell-Docs/pull/3605/commits/faca359aa06c0549cd8fad9c89c067022f5b050a)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 57 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[faca359](https://github.com/davidsmatlak/PowerShell-Docs/pull/3605/commits/faca359aa06c0549cd8fad9c89c067022f5b050a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-3.0&branch=pr-en-us-3605) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-4.0&branch=pr-en-us-3605) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-5.0&branch=pr-en-us-3605) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-5.1&branch=pr-en-us-3605) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-pssession/reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-6&branch=pr-en-us-3605) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901251821463285-3605%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0mubYWmj6x6%2B0tCRfNEMjTYIONAD2AKVXFxTi4S7fvI%3D&st=2019-01-25T18%3A34%3A06Z&se=2019-02-25T18%3A39%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 6]
```diff
@@ -234,18 +253,32 @@ Accept wildcard characters: False
...
+- **Unknown** Same as **Unicode**.
+- **UTF7** Uses UTF-7.
+- **UTF8** Uses UTF-8.
+- **UTF32** Uses UTF-32 with the little-endian byte order.
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases:
+Accepted values: ASCII, BigEndianUnicode, BigEndianUTF32, Byte, Default, OEM, String, Unicode, Unknown, UTF7, UTF8, UTF32
```
Comment (davidsmatlak) [inline reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the YAML Accepted values.

File: reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 7]
```diff
@@ -223,23 +254,39 @@ Accept wildcard characters: False
...
+- **Unknown** Same as **Unicode**.
+- **UTF7** Uses UTF-7.
+- **UTF8** Uses UTF-8.
+- **UTF32** Uses UTF-32 with the little-endian byte order.
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases:
+Accepted values: ASCII, BigEndianUnicode, BigEndianUTF32, Byte, Default, OEM, String, Unicode, Unknown, UTF7, UTF8, UTF32
```
Comment (davidsmatlak) [inline reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the YAML Accepted values.

File: reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 8]
```diff
@@ -252,35 +276,40 @@ Accept wildcard characters: False
...
+- **UTF7** Uses UTF-7.
+- **UTF8** Uses UTF-8.
+- **UTF32** Uses UTF-32 with the little-endian byte order.
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases:
-Accepted values: Unicode, UTF7, UTF8, ASCII, UTF32, BigEndianUnicode, Default, OEM
+Accepted values: ASCII, BigEndianUnicode, BigEndianUTF32, Byte, Default, OEM, String, Unicode, Unknown, UTF7, UTF8, UTF32
```
Comment (davidsmatlak) [inline reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the YAML Accepted values.

File: reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 9]
```diff
@@ -253,34 +277,39 @@ Accept wildcard characters: False
...
+- **UTF7** Uses UTF-7.
+- **UTF8** Uses UTF-8.
+- **UTF32** Uses UTF-32 with the little-endian byte order.
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases:
-Accepted values: Unicode, UTF7, UTF8, ASCII, UTF32, BigEndianUnicode, Default, OEM
+Accepted values: ASCII, BigEndianUnicode, BigEndianUTF32, Byte, Default, OEM, String, Unicode, Unknown, UTF7, UTF8, UTF32
```
Comment (davidsmatlak) [inline reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the YAML Accepted values.

File: reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 10]
```diff
@@ -253,33 +277,40 @@ Accept wildcard characters: False
...
+- **UTF8BOM**: Encodes in UTF-8 format with Byte Order Mark (BOM)
+- **UF8NOBOM**: Encodes in UTF-8 format without Byte Order Mark (BOM)
+- **UTF32**:  Encodes in UTF-32 format.
+- **Unknown**: The encoding type is unknown or invalid; the data can be treated as binary.
 
 ```yaml
 Type: Encoding
 Parameter Sets: (All)
 Aliases:
+Accepted values: ASCII, BigEndianUnicode, Byte, Default, OEM, String, Unicode, UTF7, UTF8, UTF8BOM, UF8NoBOM, UTF32, Unknown
```
Comment (davidsmatlak) [inline reference/6/Microsoft.PowerShell.Utility/Export-PSSession.md]: Added the YAML Accepted values.

**Pull request merged**